iFixit/react-commerce
# Add collection page related news stories

This PR adds related news stories to the collection page.

## QA

1. Visit the Vercel PR preview
2. Visit any collection page (other than `/parts` and `/tools`)
3. Verify that related news stories are rendered
4. Verify that related news stories are consistent with the collection

Closes #14
